packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -797,7 +797,7 @@ export class ProjectLifecycleManager {
    * centrally in the e2e tests, as well as notify the "pending initialization"
    * for run mode
    */
-  private onLoadError = (err: any) => {
+  onLoadError = (err: any) => {
```

Comment:
Can we not type this as `err: Error`? (or maybe there' a `CypressError` type, can't quite remember).